src/sqliteformat.cpp

```diff
@@ -153,6 +159,68 @@ SqliteFormat::~SqliteFormat()
     delete d;
 }
 
+bool SqliteFormat::selectMetadata(int *id)
+{
+    int rv = 0;
+
+    if (id)
+        *id = -1;
```

Comment:
Alternatively could return false if given null. Wouldn't make sense to call with such, this effectively doing nothing?

Reply:
You're right, I've done it like that now.